src/Sylius/Bundle/PaymentBundle/spec/Form/Type/PaymentGatewayChoiceTypeSpec.php

```diff
@@ -39,8 +39,8 @@ function it_is_a_form_type()
 
     function it_has_options(
         GatewayInterface $offlineGateway,
-        OptionsResolver $resolver,
-        RegistryInterface $gatewaysRegistry
+        GatewayRegistryInterface $gatewaysRegistry,
```

Comment:
As it's the collaborator injected into constructor it should be the first one here.